VAMobile/src/screens/BenefitsScreen/DisabilityRatingsScreen/NoDisabilityRatings/NoDisabilityRatings.test.tsx

```diff
@@ -1,37 +1,21 @@
 import 'react-native'
 import React from 'react'
-// Note: test renderer must be required after react-native.
-import 'jest-styled-components'
-import { ReactTestInstance, act } from 'react-test-renderer'
+import { screen } from '@testing-library/react-native'
 
-import { context, render, RenderAPI } from 'testUtils'
+import { context, render } from 'testUtils'
 import NoDisabilityRatings from './NoDisabilityRatings'
-import { TextView } from 'components'
 
 context('NoDisabilityRatings', () => {
-  let component: RenderAPI
-
-  let testInstance: ReactTestInstance
-
   const initializeTestInstance = () => {
-    component = render(<NoDisabilityRatings />, { preloadedState: {} })
-
-    testInstance = component.UNSAFE_root
+    render(<NoDisabilityRatings />)
   }
 
   beforeEach(() => {
     initializeTestInstance()
   })
 
```

Comment:
This block along with the `initializeTestInstance` function aren't needed anymore. You can render directly in the test

Reply:
updated